app/transforms/raw.js

```diff
@@ -0,0 +1,11 @@
+import DS from 'ember-data';
+
+export default DS.Transform.extend({
+  deserialize(serialized) {
+    return serialized;
+  },
+
+  serialize(deserialized) {
```

Comment:
we don't but it's a good practice to always implement hooks that utils are supposed to have